### ISSUE TITLE
Yarn dev-server failed

### ISSUE THREAD

**Issue by alemutati123** [hashlips-lab/nft-erc721-collection#122]
Hello Hashlips,

everything works well for me but in the Miniting-dapp, 'yarn dev-server' command keeps failing. I have attached the screenshot here. Can I know the reason why it keeps failing and how to fix this issue
![minting-dapp error](https://user-images.githubusercontent.com/101245747/158049083-615fca7b-27f7-4d60-8dc3-a4df56551d59.jpg)
?

**Comment by liarco** [comment]
Hi @alemutati123,
The error is truncated so I can't read it properly. Can you please share the complete path shown in the error?

Thank you

**Comment by alemutati123** [comment]
Hi Liarco,

I really appreciate your quick response. please see screenshot attached with the complete path shown
![minting-dapp error 2](https://user-images.githubusercontent.com/101245747/158070420-1d927f61-fa1a-469f-a000-e071ee0daec7.jpg)
.

**Comment by CaptainJoyboy** [comment]
I don't know if this has something to do but have you tried renaming your User's name? Maybe the `'` character is messing something up with the way the path are parsed at some point during the build. Maybe this is not the issue but Im sure is not a good idea to use `'` character in the names of the folders

**Comment by liarco** [comment]
I think @CaptainJoyboy may be right, thank you for pointing that out. @alemutati123 can you please let us know if renaming you user fixes the issue?

Thank you

**Comment by alemutati123** [comment]
@CaptainJoyboy, you were correct, it works well after I have my user name changed (not having ' in it). Thank you for your suggestion. Now I am stuck on hosting the minting site to the public. I have watched many channels on youtube and have not been successful yet. I am not a coder or haven't coded anything before, I am just trying to get everything from watching videos on youtube like Hashlips. Do you have any good recommendations for how to host the minting site publicly?

**Comment by alemutati123** [comment]
@liarco yes, @CaptainJoyboy  was right, the problem was with my user naming ('). It works well after I have it fixed. 

Thank you.

**Comment by frank-martinez-iest** [comment]
@alemutati123 send me an email at captainjoyboy@gmail.com. I can help 😊

**Comment by alemutati123** [comment]
@liarco, do you have any recommendations on hosting the minting site publicly? Now I am stuck on hosting the minting site to the public. I have watched many channels on youtube and have not been successful yet. I am not a coder or haven't coded anything before, I am just trying to get everything from watching videos on youtube like Hashlips. Do you have any good recommendations for how to host the minting site publicly? Do you think Daniel could make a video on that?

**Comment by CaptainJoyboy** [comment]
> @CaptainJoyboy, you were correct, it works well after I have my user name changed (not having ' in it). Thank you for your suggestion. Now I am stuck on hosting the minting site to the public. I have watched many channels on youtube and have not been successful yet. I am not a coder or haven't coded anything before, I am just trying to get everything from watching videos on youtube like Hashlips. Do you have any good recommendations for how to host the minting site publicly? 

I sent a reply from the wrong account hahah. @alemutati123 i can help you, send an email to captainjoyboy@gmail.com. I'm currently using DigitalOcean and is good.

**Comment by alemutati123** [comment]
> > @CaptainJoyboy, you were correct, it works well after I have my user name changed (not having ' in it). Thank you for your suggestion. Now I am stuck on hosting the minting site to the public. I have watched many channels on youtube and have not been successful yet. I am not a coder or haven't coded anything before, I am just trying to get everything from watching videos on youtube like Hashlips. Do you have any good recommendations for how to host the minting site publicly?
> 
> I sent a reply from the wrong account hahah. @alemutati123 i can help you, send an email to [captainjoyboy@gmail.com](mailto:captainjoyboy@gmail.com). I'm currently using DigitalOcean and is good.

I got it. I really appreciate for stepping up and willing to help. I already emailed you and this is my email - alemutati64@gmail.com 

Thanks!😊
